AcsfToolsCommands.php

```diff
@@ -286,7 +286,7 @@ public function dbDump(array $options = ['result-folder' => '~/drush-backups', '
     $result_folder = $options['result-folder'];
     // Validate syntax and Identify the sql parameters.
     $sql_parameters = isset($options['sql-dump-extra']) ? $options['sql-dump-extra'] : '';
-    if (!preg_match('/^[^ ].* .*[^ ]$/', $sql_parameters)) {
+    if (count(explode(' ', $sql_parameters)) > 1 && !preg_match('/^[^ ].* .*[^ ]$/', $sql_parameters)) {
```

Comment:
strpos is probably more efficient

Reply:
I suspect that something like that would be better:
```
if (!preg_match('/^[^ ]+( [^ ]+)*$/', $sql_parameters)) {
...
}

Reply:
Yes , this looks good to me, i have updated the PR with the changes , Please review.